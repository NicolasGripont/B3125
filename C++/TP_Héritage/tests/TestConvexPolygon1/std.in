PC pc1 1 3 2 5 5 6 7 4 5 1 2 2
PC pc1 1 2 3 4 5 6
PC pc2 1 2 3 4 7 1
PC pc3 1 2 2 3
PC pc3 1 2 3 4 5 6 7
PC pc4 1 1 5 5 5 1 1 5
HIT pc1 5 6 
HIT pc1 4 3
HIT pc1 7 5
HIT pc2 4 2
DELETE pc1
DELETE pc1
PC pc1 1 2 3 4 5 7
EXIT

