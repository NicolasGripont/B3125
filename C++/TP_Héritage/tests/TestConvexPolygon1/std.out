OK
ERR
OK
ERR
ERR
ERR
YES
YES
NO
YES
OK
ERR
OK